src/animation/AnimationMixer.js

```diff
@@ -428,9 +428,7 @@ AnimationMixer.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 		delete bindingByName[ trackName ];
 
-		remove_empty_map: {
```

Comment:
TBH, I've never understood the motivation behind this code. The same pattern is used in `Interpolator`.

https://github.com/mrdoob/three.js/blob/dev/src/math/Interpolant.js#L45

Why would you code this way?

Reply:
It is a way of writing goto statements in javascript. It is ill-advised to use label according to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label):

> Note: Labeled loops or blocks are very uncommon. Usually, function calls can be used instead of loop jumps.

Here it is used to jump over the delete statement when the for in loop has no items to loop over. It does this by using a label statement(`remove_empty_map:`) with a block statement (`{ ... }`).

Reply:
@Mugen87 A more direct translation without labels is:
```
var remove_empty_map = false;
for ( var _ in bindingByName ) {
    remove_empty_map = true;
    break;
}
if (remove_empty_map){
    delete bindingsByRoot[ rootUuid ];
}
```

Reply:
In any event, the new code is much more readable! Thanks for resolving this 👍

Reply:
Indeed!